### ISSUE TITLE
Change Post-processor script for P799 statements in BIB

### ISSUE THREAD

**Issue by faulhaber** [PhiloBiblon/philobiblon-ui#95]
The post-processor must leave P799 statements unchanged in BIB.
When I export the QS file and look at it directly, without post-processing, I see three  P799 statements for every record in BIB, one for Q49620 (STATUS_QNUMBER), one for Q14 (Philobiblon record created), and one for Q15 (PhiloBiblon record update).
Thus for bibid 1004:
Q14960	P799	Q49620Q14960	P799	Q14	P106	+1985-07-10T00:00:00Z/11	P721	"FAULHABER"Q14960	P799	Q15	P106	+2023-05-14T00:00:00Z/11	P721	"FAULHABER"
However, the post-processed file only shows the two P799 statement for (Q14) "PhiloBiblon record created" and (Q15) "PhiloBiblon record update":

Q14960	!P799	Q14	P106	+1985-07-10T00:00:00Z/11	P721	"FAULHABER"Q14960	!P799	Q15	P106	+2023-05-14T00:00:00Z/11	P721	"FAULHABER"

In other words the post processor deletes "Q14960 P799 Q49620" . This is the statement that captures Q49620(STATUS_QNUMBER).
I.e., the post-processing script must be changed to leave the P799 statements as they are.

**Comment by jmformenti** [comment]
Duplicates https://github.com/PhiloBiblon/philobiblon-private/issues/124
